addons/metallb/metallb.yaml

```diff
@@ -33,12 +34,20 @@ spec:
           - name: default
             protocol: layer2
             # configure addresses for your network
-            addresses: []
+            addresses:
+            - 172.17.1.200-172.17.1.250
```

Comment:
Should it be hardcoded to this? it seems this is meant to particularly support our CI environment yes?

If so, I would recommend that we provide overrides for this in the tests, specifically in that environment rather than in the defaults for the addon.

Reply:
@alejandroEsc see if you like https://github.com/mesosphere/kubernetes-base-addons/pull/19/commits/cf8c0d7735e28c15d65e2173bfe39c063a655029 or not, this would be my suggestion